modelskill/plotting/_scatter.py

```diff
@@ -257,6 +257,17 @@ def _scatter_matplotlib(
 ) -> matplotlib.axes.Axes:
     fig, ax = _get_fig_ax(ax, figsize)
 
+    N_POINTS_MIN = 50  # TODO what is a good number here?
+    show_qq = True
+
+    if len(x) < 3:
+        raise ValueError("Not enough data to plot")
```

Comment:
Why? I know it is a pretty boring plot but I would assume that 1 should be the minimum?

Reply:
write (at least 3 required)